src/helpers.js

```diff
@@ -19,19 +26,41 @@ export const createLink = (href, text) => (
   </a>
 );
 
-export const createEligibilityTooltip = (eligibility) => {
+export const populateEligibilityColumn = (eligibility) => {
   return eligibility.tooltip ? (
-    <Tooltip content={eligibility.tooltip}>
-      <span>{eligibility.title}</span>
-    </Tooltip>
+    <Popover
+      distance={0}
```

Comment:
Perhaps we could leave the default (25) value? This looks visually better if the popover is not that close to the "Not Eligible" text:

<img width="736" alt="image" src="https://github.com/user-attachments/assets/c01ed132-337e-4c69-9ef3-61730a8056ad">

Reply:
I'll compromise on 10 :smile: It seems too far away from the column when expanded out.